### PR TITLE
QAG2-107 "Create Custom Listener"

### PR DESCRIPTION
**Pull request by EmmLazyan**
"Create Custom Listener"

### CODE REVIEW

File: common_/utilities_/customListener.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+from selenium.webdriver.support.events import AbstractEventListener
+from common_.utilities_.customLogger import logger
+
+class MyListener(AbstractEventListener):
+    # def before_navigate_to(self, url, driver):
```
Comment (satikmkhitaryan) [inline common_/utilities_/customListener.py]: delete all comments

**Review by satikmkhitaryan — COMMENTED** [review]
delete all commented lines

File: common_/utilities_/customListener.py (hunk 1 above)
Comment (EmmLazyan) [reply common_/utilities_/customListener.py]: The commented lines are deleted.

**Review by satikmkhitaryan — APPROVED** [review]
ready to merge

**Pull request merged**